currency_rate_update/migrations/9.0.1.1/post-migration.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+
+
+def migrate_currency_rate_update(env):
+    """ Update field value because service selection field has changed """
+    if not openupgrade.is_module_installed(env.cr, 'currency_rate_update'):
```

Comment:
Why putting this condition? This is not going to be executed if being installed for the first time thanks to decorator `@openupgrade.migrate` (which basically checks if version is None).